src/tracer.h

```diff
@@ -34,10 +34,26 @@
 #include <sstream>
 #include <string>
 #include <unordered_map>
+
+#ifndef _WIN32
+#include "opentelemetry/exporters/otlp/otlp_http_exporter_factory.h"
+#include "opentelemetry/nostd/shared_ptr.h"
+#include "opentelemetry/sdk/trace/processor.h"
+#include "opentelemetry/sdk/trace/simple_processor_factory.h"
+#include "opentelemetry/sdk/trace/tracer_provider_factory.h"
+#include "opentelemetry/trace/provider.h"
+namespace otlp = opentelemetry::exporter::otlp;
+namespace otel_trace_sdk = opentelemetry::sdk::trace;
+namespace otel_trace_api = opentelemetry::trace;
+#endif
+
 #include "triton/core/tritonserver.h"
 
 namespace triton { namespace server {
 
+using TraceConfig = std::vector<std::pair<std::string, std::string>>;
```

Comment:
Instead of a vector of string -> string pair, how about a class/struct holding all the config settings?

Reply:
It's a good idea and can be introduced in a separate PR